tacs/problems/modal.py

```diff
@@ -372,6 +370,8 @@ def _updateAssemblerVars(self):
 
         self.assembler.setDesignVars(self.x)
         self.assembler.setNodes(self.Xpts)
+        # Make previous auxiliary loads are removed
+        self.assembler.setAuxElements(None)
```

Comment:
What's the reason for this addition?

Reply:
Other TACS problems (StaticProblem/TranientProblem) may have set an AuxElem loads in a previous call. This line makes sure that AuxElems is empty at the begining of each ModalProblem analysis, since we have no use for them